tests/osaka/eip7692_eof_v1/eip7873_tx_create/test_txcreate_failures.py

```diff
@@ -777,13 +767,14 @@ def test_reentrant_txcreate(
                 0: compute_eofcreate_address(contract_address, 0),
                 1: 0,
             }
-        )
+        ),
+        compute_eofcreate_address(contract_address, 0): Account(
```

Comment:
I think we should add a `Op.SSTORE(0, Op.ADD(Op.SLOAD(0), 1))` to the initcode to guarantee we didn't execute anything.

Same for the other reentrant case in this PR with EOFCREATE.